azure/template.json

```diff
@@ -1034,7 +774,7 @@
                         "value": "AZURE_IP-"
                     },
                     "ipAddresses": {
-                        "value": "[reference('app-service').outputs.possibleOutboundIpAddresses.value]"
+                        "value": "[reference('app-service-and-containers').outputs.appSerivceIpAddresses.value]"
```

Comment:
Looks like there is a typo here.

Reply:
That is indeed a typo, but one I carried through in the code so the functionality still worked, but I have now submitted a correction for it.